openmetadata-ui/src/main/resources/ui/src/components/BlockEditor/Extensions/image/EmbedLinkElement/EmbedLinkElement.tsx

```diff
@@ -73,9 +79,15 @@ const EmbedLinkElement: FC<ImagePopoverContentProps> = ({
               {t('label.delete')}
             </Button>
           </Space>
-          <Button htmlType="submit" type="primary">
-            {t('label.embed-file-type', { fileType })}
+
+          <Button type="link" onClick={() => onPopupVisibleChange(false)}>
+            {t('label.close')}
           </Button>
+          {isAssetsUrl ? null : (
```

Comment:
```suggestion
          {!isAssetsUrl && (
```

Reply:
It will not make much difference, but it will decrease code readability